lib/gooddata/models/process.rb

```diff
@@ -171,7 +171,7 @@ def deploy_from_appstore(path, options = { :client => GoodData.client, :project
         fail ArgumentError, 'options[:name] can not be nil or empty!' if deploy_name.nil? || deploy_name.empty?
 
         verbose = options[:verbose] || false
-        puts HighLine.color("Deploying #{path}", HighLine::BOLD) if verbose
+        GoodData.logger.info(HighLine.color("Deploying #{path}", HighLine::BOLD)) if verbose
```

Comment:
the Highline doesn't make sense any more if the string is passed to the logger

Reply:
Highline removed.